package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kriya",
-  "version": "2.6.1",
+  "version": "2.6.2",
```

Comment:
this is more a `2.7.0` I think since : 
 - API is changed
 - code doesn't exists anymore (and it should be a 3.0.0 because of that)

Reply:
code is now obvious, but not break the api. but ok for `2.7.0`